cou/steps/plan.py

```diff
@@ -67,7 +76,12 @@ async def generate_plan(analysis_result: Analysis) -> UpgradeStep:
         description="Control Plane subordinate(s) upgrade plan",
         target=target,
         filter_function=lambda app: isinstance(
-            app, (OpenStackSubordinateApplication, OpenStackAuxiliarySubordinateApplication)
+            app,
+            (
+                OpenStackSubordinateApplication,
+                OpenStackAuxiliarySubordinateApplication,
+                OvnSubordinateApplication,
+            ),
         ),
     )
     plan.add_step(control_plan_subordinate_upgrade_plan)
```

Comment:
Typo (but unrelated to your submission)
```suggestion
    plan.add_step(control_plane_subordinate_upgrade_plan)
```